_books/ion-1-1/src/macros.md

```diff
@@ -7,9 +7,31 @@ This means that Ion readers and writers often spend substantial resources proces
 Consider this example excerpt from a webserver's log file:
 
 ```ion
-{method: GET, statusCode: 200, status: "OK", protocol: https, clientIp: ip_addr::"192.168.1.100", resource: "index.html"}
```

Comment:
🗺️ These lines were long enough that they required a horizontal scroll bar. I pretty-printed them so readers can see the full volume of data without having to scroll.